src/t2iapi/operation/service.proto

```diff
@@ -29,4 +29,11 @@ service OperationService {
     rpc SetOperatingMode (t2iapi.operation.SetOperatingModeRequest)
         returns (BasicResponse);
 
+    /*
+    Modify the pm:AbstractOperationDescriptor with the given handle such that the @InvocationEffectiveTimeout is set to
+    the requested value.
+    The manipulated descriptor shall be persistent until a next manipulation call or an SDC operation invocation. If
+    the device is not able to maintain the static descriptor, it shall return RESULT_NOT_SUPPORTED.
+    */
+    rpc ModifyInvocationEffectiveTimeout (ModifyInvocationEffectiveTimeoutRequest) returns (BasicResponse);
```

Comment:
after changing the interface, maybe rename to SetInvocationEffectiveTimeout ??

Reply:
done - rpc is reworked